merged-packages/eth-json-rpc-provider/package.json

```diff
@@ -35,6 +35,15 @@
     "@types/jest": "^27.4.1",
     "depcheck": "^1.4.3",
     "deepmerge": "^4.2.2",
+    "eslint": "^8.44.0",
```

Comment:
Must have missed this the first time. Do we need these here? I think these are already in `package.json`.

Reply:
Good call! These are removed in https://github.com/MetaMask/core/pull/1766, but I forgot that I shouldn't merge that one first.

Reply:
Fixed!